Tests/Functional/ServiceInstantiationTest.php

```diff
@@ -24,17 +24,17 @@ public function testHttpClient()
     {
         static::bootKernel();
         $container = static::$kernel->getContainer();
-        $this->assertTrue($container->has('httplug.client'));
-        $client = $container->get('httplug.client');
```

Comment:
i think we should make the `httplug.client` service public too, it seems correct to me to reference it by that name.

Reply:
According to the current configuration it is configured as private. I can restore this test, but then I would also change the `public=false` in the service configuration to make clear that this id can be used.